src/Microsoft.VisualStudio.Composition/Configuration/SerializationContextBase.cs

```diff
@@ -141,6 +155,16 @@ protected internal void FinalizeObjectTableCapacity()
             }
         }
 
+        protected virtual void Dispose(bool disposing)
+        {
+            if (disposing)
+            {
+#if TRACESERIALIZATION || TRACESTATS
+                this.trace.Dispose();
+#endif
+            }
+        }
+
         protected SerializationTrace Trace(string elementName, bool isArray = false)
```

Comment:
Should this be changed to use `TypeRefFlags` once that PR is in?

Reply:
What are you referring to? The code location of this comment doesn't seem relevant to `TypeRefFlags`.

Reply:
I guess I figured that the isArray stuff was related to `TypeRefFlags` from #85 since that used to only have isArray. My thinking was that since the `Trace` method accepts isArray is should be expanded to instead take the more general `TypeRefFlags` parameter.

Reply:
No, this is a completely different `isArray` flag. This wants to know if literally we're serializing an array, which might be of bool or any other type. `TypeRefFlags` is about serializing a single `TypeRef` element.